SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -732,12 +732,23 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
         StrainMag = PerturbedStrainMag(ScalarVar_i[0]);
       }
 
-      su2double pk = Eddy_Viscosity_i * pow(StrainMag, 2) - 2.0 / 3.0 * Density_i * ScalarVar_i[0] * diverg;
-      pk = max(0.0, min(pk, 20.0 * beta_star * Density_i * ScalarVar_i[1] * ScalarVar_i[0]));
+      /*--- Definition of production P according to SSTm approximation P = mu_t S^2 ---*/
 
-      const su2double VorticityMag = GeometryToolbox::Norm(3, Vorticity_i);
-      const su2double zeta = max(ScalarVar_i[1], VorticityMag * F2_i / a1);
-      su2double pw = alfa_blended * Density_i * max(pow(StrainMag, 2) - 2.0 / 3.0 * zeta * diverg, 0.0);
+      /*--- original SST2003 model as reference (including divergence terms) ---*/
+      // su2double pk = Eddy_Viscosity_i * (pow(StrainMag, 2) - 2.0 / 3.0 * diverg * diverg) - 2.0 / 3.0 * Density_i * ScalarVar_i[0] * diverg;
+      /*--- SST2003m model, neglecting divergence terms ---*/
+      su2double pk = Eddy_Viscosity_i * pow(StrainMag, 2);
+      su2double pw = (alfa_blended * Density_i) * pk;  
+
+      /*--- Production limiter for k (and not for w) according to SST2003m ---*/
+      pk = min(pk, 10.0 * beta_star * Density_i * ScalarVar_i[1] * ScalarVar_i[0]);
```

Comment:
The limiter applies to both terms. See https://turbmodels.larc.nasa.gov/sst.html second paragraph in Menter SST Two-Equation Model from 2003

Reply:
Yes, I now found a direct reference from Menter where he corrected this: http://dx.doi.org/10.1080/10618560902773387